### ISSUE TITLE
[Adaptive Cards> FlightItinerary.json]: Passengers name is not defined inside 'dl' list tag.

### ISSUE THREAD

**Issue by Rishikesh6521** [microsoft/AdaptiveCards#9235]
### Target Platforms

UWP

### SDK Version

1.0.3.0

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Application: Adaptive Cards Visualizer
OS Version: 25H2 (OS Build 26200.5670) 
App Version: 1.0.3.0
Screen Reader: Narrator/NVDA

**Repro steps:**

1. Open the Adaptive Card Visualizer application.
2. Turn on the Narrator through Ctrl+ Windows+ Enter key and press caps lock +space bar key to turn on the scan mode.
3. Navigate to the 'FlightItinerary.json' through tab key and activate it.
4. Now, navigate to 'Passengers' name through up/down arrow key.
5. Observe the issue.

**Actual Result:**
'Passengers' name such as 'Sarah Hum', 'Jeremy Goldberg' and 'Evan Litwak' are not defined inside 'dl' list tag.
Screen reader is only announcing as **'Sarah Hum'**.

**Expected Result:**
'Passengers' name such as 'Sarah Hum', 'Jeremy Goldberg' and 'Evan Litwak' should be defined inside 'dl' list tag.
Screen reader should properly announce as **'Sarah Hum 1 of 3'** etc.

**User Impact:**
Screen reader user will get impacted as user will not be able to know that how many passengers names are present which create difficulty in making proper track of the content.

Have feedback to share on Bugs? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

![Image](https://github.com/user-attachments/assets/50b85ceb-83b9-4699-a8ef-02e90b7a1a22)

https://github.com/user-attachments/assets/79b4f1b5-7fc0-43d6-9d45-20669ff79e38

### Card JSON

```JSON
N/A
```
